chapter_linear-regression/synthetic-regression-data.md

```diff
@@ -211,33 +212,12 @@ def get_tensorloader(self, tensors, train, indices=slice(0, None)):
         return torch.utils.data.DataLoader(dataset, self.batch_size,
                                            shuffle=train)
     if tab.selected('jax'):
-        # Use PyTorch Dataset and Dataloader
+        # Use Tensorflow Datasets & Dataloader
```

Comment:
Merge line 215--216 as a single paragraph of comments. E.g.,

(max 79 chars per line)
```
# Use Tensorflow Datasets & Dataloader. JAX or Flax do not provide
# any dataloading functionality
```